src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -864,7 +864,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Private Function CheckSymbolLocationsAgainstSyntax(symbol As NamedTypeSymbol, nodeToCheck As VisualBasicSyntaxNode) As NamedTypeSymbol
             For Each location In symbol.Locations
-                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan.Start) Then
+                If location.SourceTree Is Me.SyntaxTree AndAlso nodeToCheck.Span.Contains(location.SourceSpan) Then
```

Comment:
I haven't added a test yet, and there don't seem to be _any_ tests for this, at least not that I could find. If somebody can point me in the right direction, I'll gladly add a few.